ksql-engine/src/test/java/io/confluent/ksql/KsqlContextTest.java

```diff
@@ -131,7 +141,7 @@ public void setUp() {
         serviceContext,
         SOME_CONFIG,
         ksqlEngine,
-        (ec, sc) -> InjectorChain.of(schemaInjector, topicInjector));
+        (ec, sc) -> InjectorChain.of(new PropertyOverrideInjector(), schemaInjector, topicInjector));
```

Comment:
This is a unit test. As such it should use mocks and assert the mocks are called. Not use specific classes and assert they are working correctly. Such tests duplicate tests for the specific class.

I'd actually probably change this class to mock the injector chain and test its called and the result is what is executed, etc.